CONTRIBUTING.rst

```diff
@@ -59,7 +59,7 @@ Coding Conventions
 * Functions should do one thing.
 * Early pull requests and code reviews.
 * Early architecting/design. Code reviews can happen before any code has been written.
-* Use a consistent character width of 120.
+* Use a consistent character width limit of 120.
```

Comment:
Minor, but this sounds strange to me. Shouldn't it be either "character limit" or "line width"?